packages/node_modules/@node-red/editor-client/src/js/ui/actions.js

```diff
@@ -1,47 +1,78 @@
 RED.actions = (function() {
     var actions = {
 
-    }
+    };
 
-    function addAction(name,handler) {
+    function addAction(name,handler,options) {
         if (typeof handler !== 'function') {
             throw new Error("Action handler not a function");
         }
         if (actions[name]) {
             throw new Error("Cannot override existing action");
         }
-        actions[name] = handler;
+        actions[name] = {
+            handler: handler,
+            options: options,
+        };
     }
     function removeAction(name) {
-        delete actions[name];
+        delete actions[name].handler;
```

Comment:
Removing the action should remove `actions[name]` not just the handler property
```suggestion
        delete actions[name];
```